src/sagas/lists.js

```diff
@@ -12,6 +12,11 @@ import { getQuery, pushQuery } from './helpers/url';
 import { fetchCollectionsByIDs, fetchUnitsByIDs } from './mdb';
 import { fetchViewsByUIDs } from './recommended';
 
+const endpointByNamespace = {
+  [PAGE_NS_LESSONS]: Api.lessons,
+  [PAGE_NS_EVENTS]: Api.events
```

Comment:
You area adding events to fetch by redux namespace?

Reply:
its not redux namespace its page namespace (section), maybe better separate saga function instead add _ifs_